frontend/src/pages/SubmitPunchlinePage/index.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import styles from "./style.module.css";
+import Dropdown from "../../components/Dropdown";
+import PlayerList from "../../components/PlayerList";
+import ProgressBar from "../../components/ProgressBar";
+import PunchlineCard from "../../components/PunchlineCard";
+import Button from "../../components/Button";
+
+const ChoosePunchlinePage = () => {
+  const memoryHistory = useHistory();
+
+  const dummyPunchlines: {
+    id: string;
+    text: string;
+    status: "available" | "selected" | "submitted";
+    new?: boolean;
+  }[] = [
+    {
+      id: "1",
+      text: "Me time.",
+      status: "available",
+      new: true,
+    },
+    {
+      id: "2",
+      text:
+        "Looking in the mirror, applying lipstick, and whispering “tonight, you will have sex with Tom Cruise.”",
+      status: "available",
+    },
+    {
+      id: "3",
+      text: "The violation of our most basic human rights.",
+      status: "available",
+    },
+    {
+      id: "4",
+      text:
+        "Getting married, having a few kids, buying some stuff, retiring to Florida, and dying..",
+      status: "available",
+    },
+    {
+      id: "5",
+      text: "Dark and mysterious forces beyond our control.",
+      status: "available",
+    },
+    {
+      id: "6",
+      text: "Not vaccinating my children because I am stupid.",
+      status: "available",
+    },
+    {
+      id: "7",
+      text: "Rap music.",
+      status: "available",
+    },
+    {
+      id: "8",
+      text: "Listening to her problems without trying to solve them.",
+      status: "available",
+    },
+    {
+      id: "9",
+      text: "Preteens.",
+      status: "available",
+    },
+    {
+      id: "10",
+      text: "Alcoholism.",
+      status: "available",
+    },
+  ];
+
+  const [punchlineSelected, setPunchlineSelected] = useState("");
+  const [punchlineSubmitted, setPunchlineSubmitted] = useState("");
+  const [punchlines] = useState(dummyPunchlines);
+
+  const selectPunchline = (text: string) => {
+    if (punchlineSubmitted) {
+      return;
+    }
+    if (text === punchlineSelected) {
+      setPunchlineSelected("");
+    } else {
+      setPunchlineSelected(text);
+    }
+  };
+
+  return (
+    <>
+      <Dropdown
```

Comment:
![image](https://user-images.githubusercontent.com/54158903/117537840-9eee6900-b057-11eb-964d-acb964726c5b.png)


The styling isn't quite right on the dropdown

Reply:
Yep, I noted this in the pull request comment - will return to this tomorrow as it can be fixed in isolation